### ISSUE TITLE
Registering two connectors

### ISSUE THREAD

**Issue by sebplorenz** [Mobility-Data-Space/mobility-data-space#19]
We tried to register two IDS connectors at https://ids.broker.test.mobilitydataspace.io/infrastructure, but unfortunately only one of the two connectors can be registered at any given time. Also, both the connectors somehow are interconnected to each other. We are not entirely sure why this behavior is observed, kindly advise.

**Comment by sebplorenz** [comment]
Hi, can you be more specific about "only one of the two connectors can be registered at any given time." Do the connectors have different IDs? If both connectors have different IDs then they are both visible.
What means "somehow are interconnected to each other"? What is the behavior you observe?

**Comment by sachinkumarUI** [comment]
Hi,
The two connectors do have different ids, but are being running from a same machine sharing the same IP address. 
What I mean by interconnected connector is that, lets say that I have a connector registered at the broker, now i can deregister this connector from the broker using the secondary connector

**Comment by sachinkumarUI** [comment]
Hi,
Resolved the issue by creating a different connectorId
I was expecting it to be automatically created, but in the test env we can assign any accepted value.
we managed to do it in python using uuid and md5 library. below are the steps we followed
1.  Create a a new connector id
import uuid
import hashlib

m = hashlib.md5()
m.update(connector_name.encode('utf-8'))
connectorId = str(uuid.UUID(m.hexdigest()))

2. update the config.json file with the newly created connectorId
